docs/gitlab/gitlab-gcp.md

```diff
@@ -294,7 +213,7 @@ _There is [current and ongoing effort](https://github.com/crossplaneio/gitlab-co
 We will use a convenience script for this purpose.
```

Comment:
can we please update the comment above to "There is an ongoing effort to improve secret consumption from GitLab such that secrets produced by Crossplane controllers are directly consumed without transformation"